frontend/src/reducer/publicationPage.js

```diff
@@ -13,10 +13,10 @@ import {
 import { serializeContentProps } from './workspaceContentList.js'
 import { serialize } from 'tracim_frontend_lib'
 
-function sortByModifiedDate (arrayToSort) {
+function sortByModifiedDate (arrayToSort) { // TODO MARIE inverser ici
```

Comment:
```suggestion
function sortByModifiedDate (arrayToSort) {
```
(todo is already done)